server/src/main/java/org/elasticsearch/common/lucene/search/FilteredCollector.java

```diff
@@ -34,6 +34,11 @@ public FilteredCollector(Collector collector, Weight filter) {
         this.filter = filter;
     }
 
+    /** Return the internal collector. */
+    public Collector getDelegate() {
```

Comment:
are these only used in the reduce method? it is not entirely clear to me if that reduce method is needed and so if these are needed too.

Reply:
I know reduce is called, my point is that in a single threaded collector manager there is nothing to reduce and the single collector will set what it has to set to the search context and be done with it?